SignalMessaging/ViewControllers/OWSNavigationController.m

```diff
@@ -150,7 +150,9 @@ - (void)navBarCallLayoutDidChangeWithNavbar:(OWSNavigationBar *)navbar
 
 - (UIStatusBarStyle)preferredStatusBarStyle
 {
-    if (OWSWindowManager.sharedManager.hasCall) {
+    if (!CurrentAppContext().isMainApp) {
```

Comment:
There are several changes like this where we verify that it's the main app before checking if windowManager.hasCall. Why is that? AFAIK the share extension *can't* have a call?

Reply:
PTAL @michaelkirk-signal 

I found a simpler way to fix these issues in the SAE.